libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/AbstractPluggableVisualization.tsx

```diff
@@ -287,4 +288,9 @@ export abstract class AbstractPluggableVisualization implements IVisualization {
             return resultItems;
         }, []);
     }
+
+    public abstract modifyInsightForDrillDown(
```

Comment:
I think this name is misleading. – it suggests the first argument will be modified, which is not true, right? How obout something like `getInsightWithDrillDownApplied`?